content/contribute/toBabylon/HowToContribute.md

```diff
@@ -958,68 +958,70 @@ This will run all tests that have the word "material" in their name.
 
 ### Visualization tests
 
-Run visualization tests using `npm run test:visualization` in the main directory.
+Run all visualization tests (WebGL1, WebGL2 and WebGPU) using `npm run test:visualization` in the main directory.
 
-#### Configuring the visualization tests
+#### Before your first test run
 
-Visualization tests are running using puppeteer, which is an interface to control Chrome or Firefox on Node. The test will run in a browser and will generate a report if any test fails. The browser selected is the local chromium installed together with puppeteer. If you want the tests to run in a different browser you can customize the puppeteer configuration at `jest-puppeteer.config.js` in the main directory. Some of the options are:
+The tests are running using playwright.
+If you haven't already installed playwright's browsers, please do so by running `npx playwright install`.
 
-```javascript
-module.exports = {
-  launch: {
-    dumpio: false, // should we see logs?
-    timeout: 30000, // timeout of 30 seconds
-    headless: false, // true to run headless
-    product: browser, // can be either chrome or firefox
-    ignoreHTTPSErrors: true, // should SSL issues be automatically resolved, in case the files are hosted locally on self-signed SSL certificate
-    devtools: true, // Should the dev tools open automatically
-    args: browser === "chrome" ? chromeFlags : firefoxFlags, // additional arguments for the browser, for example enabling garbage collection
-    executablePath: "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe", // choose the chrome (or firefox) to use
-  },
-  browserContext: "default",
-};
-```
+#### Running all tests
 
-Here is the list of all parameters - [https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#puppeteerlaunchoptions](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#puppeteerlaunchoptions)
+To run all visualization tests run `npm run test:visualization` in the main directory.
 
-Puppeteer opens a different context than the node context we are at when running the tests, so all code needed should either be pre-injected (using the Babylon server, for example) or sent during code evaluation. A simple example:
+If you want to run only one specific engine, add the following:
 
-```javascript
-const random = await page.evaluate((aRandomNumber) => {
-  return aRandomNumber * Math.random();
-}, Math.random());
+```shell
+npm run test:visualization -- --engine="webgl2"

```

Comment:
Is it not `--project` instead of `--engine` now?

Reply:
```suggestion
npm run test:visualization -- --project="webgl2"
```

Reply:
Thanks!